src/deploy-tes-on-azure/KubernetesManager.cs

```diff
@@ -51,9 +52,11 @@ internal class KubernetesManager
         private string valuesTemplatePath { get; set; }
         private string helmScriptsRootDirectory { get; set; }
         public string TempHelmValuesYamlPath { get; set; }
+        private AzureEnvironment azEnv;
 
-        public KubernetesManager(Configuration config, AzureCredentials credentials, CancellationTokenSource cts)
+        public KubernetesManager(AzureEnvironment env, Configuration config, AzureCredentials credentials, CancellationTokenSource cts)
```

Comment:
this shouldn't be necessary, the AzureEnvironment should already be available in the "configuration"

Reply:
Already fixed by:
```csharp
this.azEnv = AzureEnvironment.FromName(config.AzureName); 
```
Meanwhile, set KubernetesManager class property azEnv to readonly.